idp/credentials/shib-idp/secrets.properties

```diff
@@ -0,0 +1,14 @@
+# This is a reserved spot for most properties containing passwords or other secrets.
```

Comment:
Same with this, didn't see it in the old config, but looks like it is needed.  You may want to add a note in this file that these are used for testing purposes only, not production use.

Reply:
Done.